packages/ontario-design-system-component-library-react/tsconfig.json

```diff
@@ -1,13 +1,17 @@
 {
 	"compilerOptions": {
 		"allowUnreachableCode": false,
+		"paths": {
+			"@ongov/*": ["packages/*"]
+		},
+		"baseUrl": ".",
 		"allowSyntheticDefaultImports": true,
 		"declaration": true,
 		"emitDecoratorMetadata": true,
 		"experimentalDecorators": true,
 		"esModuleInterop": true,
 		"lib": ["dom", "es2015"],
-		"module": "es2015",
+		"module": "esnext",
 		"moduleResolution": "node",
```

Comment:
I'll leave this link about these options here, in case it helps as we were talking about these options the other day.  https://www.typescriptlang.org/docs/handbook/modules/theory.html#module-resolution-is-host-defined